wiki/Accuracy/id.md

```diff
@@ -0,0 +1,76 @@
+
+Di osu!standard, akurasi dikalkulasi dengan menimbang penilaian yang diperoleh dari setiap hit obyek berdasarkan nilainya dan dibagi dengan jumlah maksimum yang mungkin.
+
+Referensi untuk satu hit lingkaran:
+
+```
+300 -> 300 / 300 = 1   = 100.00%
+100 -> 100 / 300 = 1/3 =  33.00%
+50  ->  50 / 300 = 1/6 =  16.67%
+0   ->   0 / 300 = 0   =   0.00%
+```
+
+### ![](/wiki/shared/mode/taiko.png) osu!taiko
+
+![Akurasi = 0.5(jumlah GOOD + jumlah GREAT) / (jumlah BAD + jumlah GOOD + jumlah GREAT)](img/accuracy_taiko.png "Formula akurasi untuk osu!taiko")
+
+Di osu!taiko, akurasi dikalkulasikan dengan mengambil jumlah akurasi not dibagi dengan jumlah not. Akurasi not adalah sebagai berikut: sebuah GREAT (良) dihitung sebagai 100%, GOOD (可) sebagai 50% (sebagian), dan MISS/BAD (不可) sebagai 0% (yang juga memutus combo). Drum roll dan pemutar tidak mempengaruhi akurasi.
+
+### ![](/wiki/shared/mode/catch.png) osu!catch
+
+![Akurasi = (jumlah tetesan kecil + jumlah tetesan + jumlah buah) / (jumlah tetesan kecil terlewat + jumlah tetesan terlewat + jumlah buah terlewat + jumlah tetesan kecil + jumlah tetesan + jumlah buah)](img/accuracy_catch.png "Formula akurasi untuk osu!catch")
+
+Di osu!catch, akurasi dikalkulasi dengan mengambil jumlah hit obyek tanpa spinner terambil dibagi dengan jumlah hit obyek tanpa spinner. Semua hit obyek mempunyai nilai sama, kecuali pisang, karena mereka merupakan bagian dari putaran.
+
+*Catatan untuk pengguna API: Untuk menghitung akurasi di osu!catch, Jumlah tetesan kecil terdapat dalam `count50` dan jumlah tetesan kecil terlewat terdapat dalam `countkatu`.*
+
+### ![](/wiki/shared/mode/mania.png) osu!mania
+
+![Akurasi = (50 \* jumlah 50 + 100 \* jumlah 100 + 200 \* jumlah 200 + 300 \* jumlah 300 + 300 \* jumlah MAX) / 300(jumlah 0 + jumlah 50 + jumlah 100 + jumlah 200 + jumlah 300 + jumlah MAX)](img/accuracy_mania.png "Formula akurasi untuk osu!mania")
+
+Di osu!mania, akurasi dikalkulasi mirip dengan [osu!standard](#osu!standard).
+
+## Grafik performa
+
+![Grafik performa](img/performance_graph.jpg "Grafik performa")
+
+Grafik performa adalah sebuah grafik yang menampilkan performa pemain (berdasarkan bar kehidupan) selama bermain (waktu). Informasi tambahan dapat ditampilkan dengan menunjuk cursor dalam-game di atasnya.
+
+*Catatan: Informasi tambahan hanya dapat dilihat setelah bermain sebuah beatmap atau menonton sebuah putaran ulang terekspor. Setelah keluar dari [layar hasil](/wiki/results_screen), informasi ini tidak akan tersimpan.*
+
+### Akurasi
+
+Saat menunjuk di atas grafik performa, sebuah tooltip akan ditampilkan dengan *kesalahan* dan *laju tidak stabil*.
+
+Sebab cara mod [DT](/wiki/DT) (Double Time) dan [HT](/wiki/HT) (Half Time) diimplementasikan, nilai kesalahan dan laju tidak stabil akan dikalikan dengan faktor yang sama dengan lagu. Untuk mendapat nilai asli saat bermain DT, bagi hasil dengan 1.5. Sama halnya, kalikan hasil dengan 1.33 saat bermain HT.

```

Comment:
` ... Sebab cara mod [DT](/ ... `

`cara` is kind of weird ```Sebab cara mod DT dan HT diimplementasikan, nilai kesalahan```
use `saat` ```sebab saat mod DT dan HT diimplementasikan, nilai kesalahan```

Reply:
hmm ok